util/get-link.js

```diff
@@ -5,33 +5,32 @@ const youtubeSearch = require('yt-search');
 const search = promisify(youtubeSearch);
 
 function buildUrl(topResult) {
-  return (topResult.url.includes('https://youtube.com')) ? topResult.url : 'https://youtube.com' + topResult.url;
+  return (topResult.url.includes('https://youtube.com')) ?
+    topResult.url : 'https://youtube.com' + topResult.url;
 }
 
 /**
- * This function searches youtube for given songname and returns the link of topmost result
+ * This function searches youtube for given songname 
+ * and returns the link of topmost result
  *
  * @param {String} songName name of song
  * @returns {Promise<String>} youtube link of music video
  */
-const getLink = async (songName) => {
-  try {
+// this roughly equates to a max of 120mb
+const MAX_MINUTES = 60;
+const getLink = async songName => {
+  const tryLink = async () => {
     const result = await search(songName);
-    
-    const [topResult] = result.videos;
-
-    const youtubeLink = buildUrl(topResult)
-
+    const topResult = result.videos
+      .find(video => video.seconds < (MAX_MINUTES * 60));
```

Comment:
As i mentioned in the comment. Is this the reason why it skips the entire song? can we loop to first 3 songs ? and with a 'lyrics' keyword search as well? Just giving some idea. Please handle the implementation in your preferred way :)

Reply:
hmm, ill add a comment of "no matches found for" that way we can confirm that's whats occuring,

but the way i was trying to design it was it should get the first song that is under an hour long